src/server/api/question.js

```diff
@@ -14,6 +14,7 @@ export const resolvers = {
         .map({
           value: r.row("answer_option"),
           action: r.row("answer_actions"),
+          actionData: r.row("answer_actions_data"),
```

Comment:
what are the security implications here?  Where do we make sure that `actionData` isn't accessible to texters?

Reply:
I think the answer here is to only send the display name to both admins and testers, which would mask any details from unauthorized users.  Thoughts?